helpdesk/models/partner.py

```diff
@@ -0,0 +1,36 @@
+from odoo import api, fields, models
+
+
+class Partner(models.Model):
+
+    _inherit = "res.partner"
+
+    helpdesk_ticket_ids = fields.One2many(
+        comodel_name="helpdesk.ticket",
+        inverse_name="partner_id",
+        string="Assigned tickets",
```

Comment:
I don't know if assigned is the best word, maybe it should be related? A ticket is only 'assigned' to a user.